src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/HealthCertificateValidationService.swift

```diff
@@ -62,15 +62,33 @@ final class HealthCertificateValidationService: HealthCertificateValidationProvi
 	) {
 		
 		// 1. Apply technical validation
-		let expirationDate = healthCertificate.cborWebTokenHeader.expirationTime
 		
+		let isExpired: Bool
+		let stub = DCCSignatureVerifiableStub(error: nil)
+		// DONT FORGET: pass signatures array into this function!
+		let result = stub.verify(certificate: healthCertificate.base45, with: [], and: validationClock)
```

Comment:
for me it is confusing that in our production code we use a stub to verify something. Perhaps change just the wording, but my feeling about this is bad.

Reply:
Signature service is not injected.